### ISSUE TITLE
[Windows] Private Key file as a Temporary file is accessible by every user by default

### ISSUE THREAD

**Issue by Chocaute** [dlech/KeeAgent#389]
Greetings, 

Case in point,

![Case in point.](https://github.com/dlech/KeeAgent/assets/136109086/404b14c6-34d0-4a1d-8593-954bda202ed9)

I know this is a temporary file, but doesn't this defeat the point of KeePass? 

For context, I used this feature as a step to stymie [having lots of SSH keys loaded at once](https://keeagent.readthedocs.io/en/latest/usage/tips-and-tricks.html#creating-url-overrides-to-open-a-connection).

**Comment by Gnarfoz** [comment]
Note: _SYSTEM_ is the highest privileged user on Windows. You seem to have it mixed up with _Everyone_?

**Comment by Chocaute** [comment]
> Note: _SYSTEM_ is the highest privileged user on Windows. You seem to have it mixed up with _Everyone_?

Not at all, I was just kind of miffed that one of my private key file was accessible to every user on my system (Which is me and my brother) in some temp folder. 

It's not that it's an actually issue, but shouldn't private key files be only accessible to its owner?
